src/Raven.Client/Documents/Linq/RavenQueryProviderProcessor.cs

```diff
@@ -1052,7 +1056,7 @@ private void VisitEnumerableMethodCall(MethodCallExpression expression, bool neg
                         if (expression.Arguments.Count == 1 && expression.Arguments[0].Type == typeof(string))
                         {
                             _documentQuery.OpenSubclause();
-                            _documentQuery.Where("*:*");
+                            //_documentQuery.Where("*:*");
```

Comment:
Ups, supposed to be `_documentQuery.WhereTrue()`, thought that I fixed all of those.